internal/rest/avi_obj_vsvip.go

```diff
@@ -118,8 +118,9 @@ func (rest *RestOperations) AviVsVipBuild(vsvip_meta *nodes.AviVSVIPNode, vsCach
 				vsvip.Vip = []*avimodels.Vip{}
 				vsvip.Vip = append(vsvip.Vip, vips...)
 			} else if lib.GetCloudType() == lib.CLOUD_NSXT && lib.GetVPCMode() {
-				vpcArr := strings.Split(vsvip_meta.T1Lr, "/")
-				vipNetwork := fmt.Sprintf("%s_AVISEPARATOR_%s_AVISEPARATOR_PUBLIC", vsvip_meta.Tenant, vpcArr[len(vpcArr)-1])
+				vpcArr := strings.Split(vsvip_meta.T1Lr, "/vpcs/")
```

Comment:
@abhishekbsingh Do you want this changes to be present? Rishabh's PR is taking care of this.